tests/bdd/command-line/replication.feature

```diff
@@ -1,32 +1,337 @@
-Feature: Tests for the osm2pgsql-replication script
-
-    Scenario: Replication can be initialised
+Feature: Tests for the osm2pgsql-replication script with property table
 
+    Scenario: Replication can be initialised with an osm file after import
         Given the OSM data
             """
             n34 Tamenity=restaurant x77 y45.3
             """
+        And the replication service at http://example.com/europe/liechtenstein-updates
         When running osm2pgsql pgsql with parameters
             | --slim |
 
         And running osm2pgsql-replication
             | init | --osm-file={TEST_DATA_DIR}/liechtenstein-2013-08-03.osm.pbf |
 
-        Then table planet_osm_replication_status has 1 row
+        Then table osm2pgsql_properties contains
+            | property                    | value                                           |
+            | replication_base_url        | http://example.com/europe/liechtenstein-updates |
+            | replication_sequence_number | 9999999                                         |
```

Comment:
Where does this sequence number come from?

Reply:
From the test file, see #1931.